docs/features/logforwarding/outputs/syslog-forwarding.adoc

```diff
@@ -0,0 +1,148 @@
+== Forwarding to Syslog
+
+Create a Cluster Log Forwarder instance by specifying corresponding parameters.
+
+.cluster-log-forwarder.yaml
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+  - name: syslog
+    syslog:
+      rfc: RFC3164 # <1>
+      url: http://my-syslog.com # <2>
+      severity: informational # OPTIONAL parameter can be skipped will be calculated automatically <3>
+      facility: user # OPTIONAL parameter can be skipped will be calculated automatically <4>
+      payloadKey: message # OPTIONAL parameter can be skipped will be calculated automatically <5>
+      appName: myApp # OPTIONAL parameter can be skipped will be calculated automatically <6>
+      procId: 3144 # OPTIONAL parameter can be skipped will be calculated automatically <7>
+      msgId: MsgID # OPTIONAL parameter can be skipped will be calculated automatically <8>
+      enrichment: KubernetesMinimal # <9>
+    type: syslog
+  pipelines:
+    - name: demo-logs
+      inputRefs:
+        - application
+        - infrastructure
+        - audit
+      outputRefs:
+        - syslog
+  serviceAccount:
+    name: mine
+----
+Where `url` and `rfc` parameters are mandatory other are optional
+
+. `rfc`: Syslog RFC specification:  `RFC3164` or `RFC5424`
+. `url`: an absolute URL, with a scheme. Valid schemes are: `http`, `https`, `tcp`, `tls`, `udp` and `udps`
+. `severity`: values are defined in https://tools.ietf.org/html/rfc5424#section-6.2.1
+  The value can be a decimal integer or one of these case-insensitive keywords:
+
+     Emergency
+     Alert
+     Critical
+     Error
+     Warning
+     Notice
+     Informational
+     Debug
+
+. `facility`: values are defined in https://tools.ietf.org/html/rfc5424#section-6.2.1.
+  The value can be a decimal integer. Facility keywords are not standardized,
+  this API recognizes at least the following case-insensitive keywords
+  (defined by https://en.wikipedia.org/wiki/Syslog#Facility_Levels):
+
+    kernel user mail daemon auth syslog lpr news
+    uucp cron authpriv ftp ntp security console solaris-cron
+    local0 local1 local2 local3 local4 local5 local6 local7
+
+
+. `payloadKey`: specifies record field to use as payload. This supports template syntax to allow dynamic per-event values.
+  The `payloadKey` must be a single field path encased in single curly brackets `{}`.
+  Field paths must only contain alphanumeric and underscores. Any field with other characters must be quoted.
+  If left empty, Syslog will use  the whole message as the payload.
+  Examples:
+
+  {.bar}
+  {.foo.bar.baz}
+  {.foo.bar."baz/with/slashes"}
+
+. `appName`: is APP-NAME part of the syslog-msg header.
```

Comment:
> We need a clarification here and similar in the remaining part of the doc. This is a correct statement but it reads to me like we expect a user to set this in our AP

Please update these fields to say Optional where appropriate as in the sample above.

Reply:
added here:  https://github.com/openshift/cluster-logging-operator/pull/2907/files#diff-dcb126cbdb763df54cb71b630990f555a494cc9cb5406b0fb018b157a9d5e035R38